src/rules/mergeableNamespaceRule.ts

```diff
@@ -0,0 +1,54 @@
+/**
+ * @license
+ * Copyright 2016 Palantir Technologies, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as ts from "typescript";
+import * as Lint from "../lint";
+
+export class Rule extends Lint.Rules.AbstractRule {
+    public static FAILURE_STRING = "mergeable namespace: ";
```

Comment:
Going forward, for rules that have slightly customized failure strings like this, let's create a `failureStringFactory` method instead of a constant like this. In this case, we can actually make the failure much more helpful, something like this:

```ts
public static failureStringFactory(identifier: string, locationA: ts.LineAndCharacter, locationB: ts.LineAndCharacter): string {
    return `Mergeable namespace ${identifier} found. Merge its contents with the namespace on line ${locationB.line}.`;
}
```